### PR TITLE
add js-dom task

### PR DESCRIPTION
**Pull request by frostwolm**
### js-dom

[Demo](https://frostwolm.github.io/js-dom/) |
[Code base](https://github.com/frostwolm/js-dom)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/frostwolm/js-dom/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <nav class="menu">
+      <ul class="menu__list">
+      </ul>
+    </nav>
+    <main>
+      <article class="text">
+      </article>
+    </main>
+  </div>
+  <script type="text/javascript">
```
Comment (A-Ostrovnyy) [inline submissions/frostwolm/js-dom/index.html]: Separate responsobility: HTML - marckup, CSS - styles, JS - logic. Move `<script>` into `.js` files.

File: submissions/frostwolm/js-dom/js/app.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+const contentController = {
+  init(navElem, contentElem, articlesArr = articles) {
+    if (!navElem || !articlesArr || !contentElem) {
+      console.error(`Не определен элемент навигации или элемент контента`);
+      return null;
+    }
+    this.articles = articlesArr;
+    navElem.innerHTML = ``;
+    let ulElem = document.createElement(`ul`);
```
Comment (A-Ostrovnyy) [inline submissions/frostwolm/js-dom/js/app.js]: You already have `ul` tag inside `index.html`.

File: submissions/frostwolm/js-dom/css/app.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    max-width: 100%;
+  }
+
+  .menu__list{
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .text__img{
+    float: left;
```
Comment (A-Ostrovnyy) [inline submissions/frostwolm/js-dom/css/app.css]: Don't forget about `clearfix` if you use float.

File: submissions/frostwolm/js-dom/js/app.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    });
+    contentElem.innerHTML = ``;
+    let h2Elem = document.createElement(`h2`);
+    h2Elem.textContent = article.title;
+    h2Elem.classList.add(`text__title`);
+    let imgElem = document.createElement(`img`);
+    imgElem.setAttribute(`src`, article.imgSrc);
+    imgElem.setAttribute(`alt`, article.title);
+    imgElem.classList.add(`text__img`);
+    let sectionElem = document.createElement(`section`);
```
Comment (A-Ostrovnyy) [inline submissions/frostwolm/js-dom/js/app.js]: Wrong usage `<section>` tag. [Read more.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)

**Comment by frostwolm** [comment]
Done

**Comment by A-Ostrovnyy** [comment]
Great. Good work!

**Pull request merged**